### PR TITLE
OCM-2670: Fix GitHub IDP example

### PR DESCRIPTION
**Pull request by sagidayan**
Could not find the cause of the array not being passed to terraform. 

This PR will simply make the example work.

### CODE REVIEW

File: examples/create_identity_provider/github/variables.tf [hunk 1]
```diff
@@ -24,7 +24,7 @@ variable "github_client_secret" {
   type        = string
   description = "GitHub client secret"
 }
-variable "github_orgs" {
-  type        = list(string)
-  description = "List of GitHub organisations"
+variable "github_org" {
+  type        = string
+  description = "GitHub organisation"
```
Comment (osherdp) [inline examples/create_identity_provider/github/variables.tf:29]:
```suggestion
  description = "GitHub organization"
```

**Review by osherdp — CHANGES_REQUESTED** [review]

File: examples/create_identity_provider/github/main.tf [hunk 2]
```diff
@@ -34,6 +34,6 @@ resource "rhcs_identity_provider" "github_idp" {
   github = {
     client_id     = var.github_client_id
     client_secret = var.github_client_secret
-    organizations = var.github_orgs
+    organizations = [var.github_org]
```
Comment (osherdp) [inline examples/create_identity_provider/github/main.tf:37]: wdym by saying the array isn't being passed to terraform? we should have some examples like that in assisted I think: https://github.com/openshift/assisted-test-infra/blob/ca8ab6295b6e401508d2d164058bdb2e58e4ab93/terraform_files/baremetal/main.tf#L26

Comment (sagidayan) [reply examples/create_identity_provider/github/main.tf:37]: I think its something to do with the **(OLD)** sdk version we are using. An old one.

We could not figure it out. So we will reach out to HashiCorp. But for now, we want the example to simply work.

**Review by osherdp — APPROVED** [review]

**Pull request merged**